external/downstream/pasture/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-if(ENABLE_psi4pasture)
-  ExternalProject_Add(psi4pasture
-    DEPENDS psi4_core
+if(ENABLE_pasture)
+  ExternalProject_Add(pasture
+    DEPENDS psi4-core
     GIT_REPOSITORY https://github.com/psi4/psi4pasture
     CMAKE_ARGS -DENABLE_ccsort=${ENABLE_ccsort}
                -DENABLE_transqt2=${ENABLE_transqt2}
```

Comment:
It looks like we will also need to add `-Dpybind11_DIR=${STAGED_INSTALL_PREFIX}/share/cmake/pybind11` to the `CMAKE_ARGS` section or else cmake chokes on `find_package(psi4 1.0 REQUIRED)` due to [this](https://github.com/psi4/psi4/pull/554/files#diff-720577ced4653fc45bc393ee4df2dc42R109) line in the psi4config.cmake